core/ostctools.c

```diff
@@ -57,25 +57,37 @@ void ostctools_import(const char *file, struct dive_table *divetable)
 	// Read dive number from the log
 	uc_tmp = calloc(2, 1);
 	fseek(archive, 258, 0);
-	fread(uc_tmp, 1, 2, archive);
+	if (fread(uc_tmp, 1, 2, archive) != 2) {
+		report_error(translate("gettextFromC", "Failed to read '%s'"), file);
```

Comment:
let's include this as a variable near the function  prologue to help the translation tooling:
`const char *failed_to_read = translate("gettextFromC", "Failed to read '%s'");`
and then re-use `failed_to_read`.

Reply:
we should use `failed_to_read_msg` here.